docs/app/templates/getting-started/assets.hbs

```diff
@@ -64,13 +64,12 @@ module.exports = function(defaults) {
     href="https://github.com/adopted-ember-addons/ember-cli-sass"
     target="_blank"
     rel="noopener noreferrer"
-  >ember-cli-sass</a>) as a CSS preprocessor. When it detects it upon first
-  installation of ember-bootstrap, it will install the necessary packages and
-  add an
+  >ember-cli-sass</a>) as a CSS preprocessor. Ensure that you have the Bootstrap
+  package installed and add an
   <code>@import</code>
```

Comment:
Should we still recommend using `@import`?
It seems like SASS is trying to slowly get rid of it in favor of `@use` and `@forward`.

Reply:
Bootstrap still uses `@import` in its official documentation: https://getbootstrap.com/docs/5.3/customize/sass/ I feel we should stay aligned with Bootstrap documentation in that regard.